src/SelectableBox/index.jsx

```diff
@@ -48,7 +48,7 @@ const SelectableBox = React.forwardRef(({
     tabIndex: -1,
     onChange: () => {},
     ...(type === 'checkbox' ? { ...props, isIndeterminate } : { ...props }),
-  }, null);
+  }, false);
```

Comment:
can you please explain why this change is necessary?

Reply:
`Failed prop type: The prop `children` is marked as required in `ForwardRef`, but its value is `null``

Reply:
[clarification] we reverted this change because it will be fixed by #1988